### DISCUSSION TITLE
Amplify

### DISCUSSION THREAD

**Discussion by prasadbobby** [prasadbobby/amplify#37] [Q&A] [answered]
Amplify is the React JS application where all the Materials are found in this application.
But there are minor bugs in this application. Please help me to find it out and also give some better ideas to implement in this application.

**Comment by mahendrasaikumargandham** [comment, marked as answer]
I can give you some suggestions regarding UI.
It's ok now but you need to update your media tag for more responsive @prasadbobby 
Let's discuss some more things later . :)
